frontend/app/src/components/SelectChatModal.svelte

```diff
@@ -17,6 +17,7 @@
     import SectionHeader from "./SectionHeader.svelte";
     import { _ } from "svelte-i18n";
     import { now } from "../stores/time";
+    import { usernameAndIcon } from "openchat-shared";
```

Comment:
we seem to have a few of these creeping in. The original idea was that the shared lib was only there to share things internally between various front end components but that the actual website code would only ever depend on the openchat-client project. It's easy enough to just expose whatever you need from openchat-shared via the openchat-client package.